src/templates/requests_page.hbs

```diff
@@ -1,91 +1,60 @@
-<div class="my-activities-nav">
-  <div class="container">
-    <nav class="collapsible-nav">
-      <button type="button" class="collapsible-nav-toggle" aria-label="{{t 'toggle_navigation'}}" aria-expanded="false">
-        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" focusable="false" viewBox="0 0 12 12" aria-hidden="true" class="collapsible-nav-toggle-icon chevron-icon">
-          <path fill="none" stroke="currentColor" stroke-linecap="round" d="M3 4.5l2.6 2.6c.2.2.5.2.7 0L9 4.5"/>
-        </svg>
-        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" focusable="false" viewBox="0 0 12 12" aria-hidden="true" class="collapsible-nav-toggle-icon x-icon">
-          <path stroke="currentColor" stroke-linecap="round" d="M3 9l6-6m0 6L3 3"/>
-        </svg>
-      </button>
-      <ul class="collapsible-nav-list">
-        <li class="current">
-          <a href="{{page_path 'requests'}}" aria-current="page">{{ t 'requests' }}</a>
-        </li>
-        <li>{{link 'contributions'}}</li>
-        <li>{{link 'subscriptions'}}</li>
-      </ul>
-    </nav>
-  </div>
-</div>
+<script>
+  window.addEventListener('DOMContentLoaded', (event) => {
+      const $rows = document.querySelectorAll('tr[data-href]');
```

Comment:
I'm a bit worried that this css selector might be too broad. I realize it is unlikely another table would be loaded on this page, it's still a good practice to scope behavior to specific tables instead of using a general selector that can be applied to all table rows.

I think just adding a class to the table where we expect this behavior, and then targeting the child table rows of that class would be safer.